### ISSUE TITLE
Provide a second overview plot which shows only the recent past

### ISSUE THREAD

**Issue by fangohr** [oscovida/oscovida#51]
In many regions, the big peak has passed, and the maximum sets the scale in many plots. 

Would be useful to just show the recent pass to be able to better assess effectiveness of containment measures.

The last 4 weeks? 

Might need an update to the `overview` function to allow this. Would be good if it could be done flexibly so it can be used for interactive exploration.

**Comment by kirienko** [comment]
In the beginning of the story, some early March, everyone was trying to compare countries as if the disease had started at the same time. It was interesting how countries can cope with the situation and how effective the measures they took. That's where those starting point normalization came from (I mean all those «since the first 10 cases» and so on).

But now the situation has changed, and I suppose that that comparison is no longer that interesting. People are rather talking about the current state, kind of a snapshot. That's why I'd offer the following design solution:

* by default we show the last, say, 4 weeks (looks reasonable)
* but as an option we allow to pick either
  - beginning of the story, whenever it was (those "10 first cases")
  - arbitrary date in the past (if we have data for that day of course, we can limit it with, say, 01.03.2020)

What do you guys think?

**Comment by fangohr** [comment]
The "since first 10 cases" is only used in the last two plots (plots 7 and 8 in the attached bitmap). For those - because of the log scale - I find it quite useful to have the whole duration presented; I think it helps in understanding the (time) development in that country.

For the other plots (numbers 1 to 6 in the example below), I agree that the interesting use case are:
1. see the whole history up to today (as we have it at the moment)
2. a recent period (for example 4 weeks; that's about the timescale from infections starting to be visible in deaths, so maybe even a bit l onger)
3. a flexibly chosen date range. I think often one may want to choose the start date freely and use the current date as the end date. But I can also think of situations where I want to pick start and end-date manually, for example to look at a local outbreak, or the first or second wave of the pandemic.

I agree with the suggestion to make use case 2 the default and most important data: we want to know what is happening at the moment. 
I have played with the 3 options manually some time ago. I found that just the use case 2 is often not providing enough context, and it is useful to see the data for the whole duration in addition to the last n weeks. 

If we stick to the notebook interface, this would mean for me to have the last 4 weeks shown at the top as the first figure. This could include plots 1 to 6, and 7 and 8. We could potentially change the line style (to a thicker or less transparent line?) for those data points that represent the last 4 weeks, to provide context. Or we drop plots 7 and 8 from this figure 1.

As a second figure, I could then imagine to provide the overview of the whole development in that region (potentially not showing plot 7 and 8 if they are included in figure 1).

To cover for use case 3, we could have a command for figure 2 that includes the data range that is plotted, and we could insert the strings for the current command. I.e. if today is 30 June and we want to plot the last 4 weeks, the relevant command we use to create figure 2 could be 
```
cases  = get_cases_command(COUNTRYNAME)
plot_overview(cases["02-06-2020":"30-06-2020"])
```
If we provide this example, the use case 3 can be solved by opening the notebook in Binder, and modifying that pre-populated string.

![Screenshot 2020-07-03 at 17 33 17](https://user-images.githubusercontent.com/4489119/86483005-69039880-bd53-11ea-9866-76047325048d.png)

Thoughts?

**Comment by kirienko** [comment]
In #80 I've implemented showing only the last 5 weeks, and it looks like this:

![image](https://user-images.githubusercontent.com/1487169/87192484-dd5fae00-c2f6-11ea-89d9-2a9a28630dc3.png)

I forgot to adjust the x-ticks and labels — now they looks, let's say, untidy.


For the last two graphs (_"compare plots"_) situation is different and it requires much more work: now both graphs are aligned from the left hand side. If we chop out only the last `N` days, will will have to align all graphs from the right. But the bigger problem is that after such change the graphs might be useless (the whole idea was to compare the entire trend of the pandemic).
Anyway, I'd prefer to do it as a separate issue after finishing this one.

**Comment by kirienko** [comment]
X-ticks are looking good now:

![image](https://user-images.githubusercontent.com/1487169/87203427-dc347e00-c302-11ea-9dbf-bf9f4f689c77.png)

**Comment by fangohr** [comment]
Nice. 

The y scale for blue line in last plot doesn’t seem to be right yet?

You can now see the short term changes in daily new numbers - this isn’t available in the overview plot. 

(I still think we might need both plots).

**Comment by kirienko** [comment]
@fangohr I don't quite understand your point with "both plots", so correct me if I'm wrong:

![issue_80](https://user-images.githubusercontent.com/1487169/88202557-59d48400-cc49-11ea-8ea8-4ec8a5cd52e0.png)

Is it what you want?

It looks for me that if we have both, the picture gets extremely large (and boring). And take into account that I've already zoomed out to 80% of natural size.

**Comment by fangohr** [comment]
Yes, that's what I meant, thank you!

I agree it is lots of axis, but the two sets of plots show two entirely different things: the first one is focused on the recent past. This matters if you care about current changes. This is important as the overview plots we have at the moment do not allow to see this kind of data.

The second puts this first data set in context and provides the history  of the pandemic in that area. This is also important if you want to know what the history is (and for most countries other than your own or your favourites one will not know this and needs to look at it).


In terms of practical realisation: In the notebook, I would have one command to create the first set of plots, and a second command to create the second set. (And combine this with clear headings such as : "Last 5 weeks", "Complete history". (These can be improved.) The two commands would be very similar, and would also serve as an example of commands that the user can modify in a binder session.

We could detach the country comparison plots (the two at the end) so that one needs a third command to create that? 

I am open to discussion and for ideas - maybe we can brainstorm tomorrow in the meeting.

**Comment by kirienko** [comment]
Solved via #80
